PropertyChangedAnalyzers.Test/INPC001ImplementINotifyPropertyChangedTests/CodeFix.cs

```diff
@@ -1,4 +1,4 @@
-namespace PropertyChangedAnalyzers.Test.INPC001ImplementINotifyPropertyChangedTests
+﻿namespace PropertyChangedAnalyzers.Test.INPC001ImplementINotifyPropertyChangedTests
```

Comment:
Your .editorconfig has `charset = utf-8-bom` which is causing every file to be converted on save.

Reply:
Ah, been wondering :)
It will go away once every file is saved right?

Reply:
Yes. My approach has been to rewrite all files to match the editorconfig in the same PR that changes the editorconfig, but this works too. (In my case, I'm always losing the byte order mark...)